SETUP/tests/MARCRecordTest.php

```diff
@@ -1,4 +1,6 @@
 <?php
+$relPath='../../pinc/';
```

Comment:
You shouldn't need either of these lines. `phpunit_bootstrap.php` pulls in base.inc which installs an autoloader which will pull in MARCRecord on first use of the class. These tests should always be run with the bootstrap:
```
To run them, use the `phpunit_bootstrap.php` file to pull in pre-requisits.
    phpunit --bootstrap phpunit_bootstrap.php --verbose .
```